reconcile/github_org.py

```diff
@@ -499,3 +499,10 @@ def run(dry_run):
     )
 
     runner.run()
+
+
+def early_exit_desired_state(*args, **kwargs) -> dict[str, Any]:
+    return {
+        "github_org_config": get_config(),
```

Comment:
why isn't this enough? https://github.com/app-sre/qontract-reconcile/blob/4c5749bb25ebb08741c3f1579a259801c5f53fad/reconcile/github_org.py#L95

Reply:
do you mean managed teams?

Reply:
if so, the list of managed teams is a part of the query: https://github.com/app-sre/qontract-reconcile/blob/4c5749bb25ebb08741c3f1579a259801c5f53fad/reconcile/github_org.py#L32

Reply:
yes sry i meant `managedTeams`. the query you posted is exactly used by `get_config()`

Reply:
maybe i misunderstood you question. let me revisit

Reply:
ah ok i get it. you mean "why the additional processing of `get_config` if the core information required is get result of the query. good point. i will refactor it

Reply:
see [34feef4](https://github.com/app-sre/qontract-reconcile/pull/2698/commits/34feef40903e8a03583bbeb0ddcef7fc7dae39d4)